<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>573</width>
      <height>344</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutMain">
     <item row="3" column="1">
      <widget class="QGroupBox" name="groupBoxWordSize">
       <property name="title">
        <string>Word size</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>282</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayoutWordSize">
         <item>
          <widget class="QRadioButton" name="radioButtonQWord">
           <property name="text">
            <string>QWord</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonDWord">
           <property name="text">
            <string>DWord</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonWord">
           <property name="text">
            <string>Word</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonByte">
           <property name="text">
            <string>Byte</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelOperationsList">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayoutKeyboard">
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutOperations">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutCancel">
           <item>
            <widget class="QPushButton" name="pushButtonCancel">
             <property name="text">
              <string>C</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonCalculate">
             <property name="text">
              <string>=</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutErase">
           <item>
            <widget class="QPushButton" name="pushButtonErase">
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutNormalOperations">
           <item>
            <widget class="QPushButton" name="pushButtonCE">
             <property name="text">
              <string>CE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonPlusMinus">
             <property name="text">
              <string>Â±</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonMod">
             <property name="text">
              <string>MOD</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutArythmeticOperations">
           <item>
            <widget class="QPushButton" name="pushButtonPlus">
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonMinus">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonMultip">
             <property name="text">
              <string>*</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDiv">
             <property name="text">
              <string>/</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutBitOperations">
           <item>
            <widget class="QPushButton" name="pushButtonAnd">
             <property name="text">
              <string>AND</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonNot">
             <property name="text">
              <string>NOT</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonOr">
             <property name="text">
              <string>OR</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonXOr">
             <property name="text">
              <string>XOR</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutShiftBitOperations">
           <item>
            <widget class="QPushButton" name="pushButtonRoL">
             <property name="text">
              <string>ROL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonRoR">
             <property name="text">
              <string>ROR</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonLSh">
             <property name="text">
              <string>LSH</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonRSh">
             <property name="text">
              <string>RSH</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutKeyboard">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutNumbersFirstLine">
           <item>
            <widget class="QPushButton" name="pushButton1">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton2">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton3">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutNumbersSecondLine">
           <item>
            <widget class="QPushButton" name="pushButton4">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton5">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton6">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutNumbersThirdLine">
           <item>
            <widget class="QPushButton" name="pushButton7">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton8">
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton9">
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutZeroLine">
           <item>
            <widget class="QPushButton" name="pushButtonLeftBracket">
             <property name="text">
              <string>(</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton0">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonRightBracket">
             <property name="text">
              <string>)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutHexValuesFirstLine">
           <item>
            <widget class="QPushButton" name="pushButtonA">
             <property name="text">
              <string>A</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonB">
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonC">
             <property name="text">
              <string>C</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutHexValuesSecondLine">
           <item>
            <widget class="QPushButton" name="pushButtonD">
             <property name="text">
              <string>D</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonE">
             <property name="text">
              <string>E</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonF">
             <property name="text">
              <string>F</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" rowspan="3">
      <layout class="QHBoxLayout" name="horizontalLayoutNumericSystemsLayout">
       <item>
        <widget class="QGroupBox" name="groupBoxNumeralSystems">
         <property name="title">
          <string>Numeral systems</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>261</width>
            <height>90</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayoutResultSize">
             <item>
              <widget class="QRadioButton" name="radioButtonBin">
               <property name="text">
                <string>Bin</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonHex">
               <property name="text">
                <string>Hex</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonDec">
               <property name="text">
                <string>Dec</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonOct">
               <property name="text">
                <string>Oct</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayoutResultSizeDisplays">
             <item>
              <widget class="QLabel" name="labelBin">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelHex">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelDec">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelOct">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QLabel" name="labelResult">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>403</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listWidgetMemory"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButtonFromMemoryToCalculator">
         <property name="text">
          <string>&lt;-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonMS">
         <property name="text">
          <string>MS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonMC">
         <property name="text">
          <string>MC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonMPlus">
         <property name="text">
          <string>M+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonMMinus">
         <property name="text">
          <string>M-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
